test/config_disseminate_test.go

```diff
@@ -153,3 +169,196 @@ func TestConfigDisseminate(t *testing.T) {
 		require.True(t, protoutil.IsConfigBlock(lastBlock))
 	}
 }
+
+func TestConfigTXDisseminationWithVerification(t *testing.T) {
+	// Compile Arma
+	armaBinaryPath, err := gexec.BuildWithEnvironment("github.com/hyperledger/fabric-x-orderer/cmd/arma", []string{"GOPRIVATE=" + os.Getenv("GOPRIVATE")})
+	defer gexec.CleanupBuildArtifacts()
+	require.NoError(t, err)
+	require.NotNil(t, armaBinaryPath)
+
+	// Generate the configuration with clientSignatureVerificationRequired = True
+	dir := t.TempDir()
+	configPath := filepath.Join(dir, "config.yaml")
+	numOfParties := 4
+	numOfShards := 2
+	orgNum := 1
```

Comment:
This name is a bit confusing as it is actually the id of the party which submits the TX